main/main.cpp

```diff
@@ -1399,6 +1398,11 @@ bool Main::start() {
 		}
 
 		String index_path = doc_tool.plus_file("doc/classes");
+		// Create the directory if it doesn't exist
+		DirAccess *da = DirAccess::create_for_path(index_path);
```

Comment:
Is this the "best" way to create directories in this situation? I saw a `DirAccessRef` instance in a block a few lines above, but here, I based it on the `make_dir_recursive()` implementation in https://github.com/Calinou/godot/blob/888c9b3066b170cb9d1e7171a1700b5c2b654948/core/bind/core_bind.cpp#L2260-L2271

Reply:
It seems to be done this way in various places indeed, so that seems fine.

Reply:
Though I don't know if we should always attempt to recreate the directory even if it already exists. And a similar operation would be needed for modules doc directories in the for loop above. Last I played around with this I got crashes and gave up :P